server/src/main/java/com/workshop/domain/VehicleType.java

```diff
@@ -0,0 +1,11 @@
+package com.workshop.domain;
+
+public enum VehicleType {
+    SEDAN,
+    COUPE,
+    SPORT,
+    HATCHBACK,
+    SUV,
+    CONVERTIBLE,
+    MINIVAN,
```

Comment:
I'd suggest simplifying this distinction: 

- CITY_CAR
- COMPACT
- BIG_SEDAN
- SUV
- MINIVAN
- TRUCK
- CONVERTIBLE
These would reflect differences in size better imo.

Reply:
That make sense